shell/platform/android/io/flutter/embedding/engine/renderer/FlutterRenderer.java

```diff
@@ -44,6 +46,10 @@ public class FlutterRenderer implements TextureRegistry {
   private boolean isDisplayingFlutterUi = false;
   private Handler handler = new Handler();
 
+  @NonNull
+  private final Set<TextureRegistry.OnLowMemoryListener> onLowMemoryListeners =
+      new CopyOnWriteArraySet<>();
```

Comment:
is this being accessed from different threads?

Reply:
this is still unresolved

Reply:
This one has been solved.